cfme/web_ui/__init__.py

```diff
@@ -2061,6 +2061,8 @@ def _find_in_tree(t, p=None):
 
 @fill.method((BootstrapTreeview, Sequence))
 def _fill_bstree_seq(tree, values):
+    if not values:
```

Comment:
@psav 
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/cloud_infra_common/test_vm_ownership.py#L25
https://github.com/ManageIQ/integration_tests/blob/master/cfme/configure/access_control.py#L406
When no need to change default values for product features